travvy/src/components/AttractionDetails.vue

```diff
@@ -1,23 +1,29 @@
 <template>
   <div>
-    <!-- Button to go to go back to the AttractionsList page -->
+    <div class="container">
```

Comment:
Thank you for adding a container to this page! It'll finally make it consistent with the rest of the UI as well.